From 28a8c2b2ba9e6af4da016beae1e6679015f6d591 Mon Sep 17 00:00:00 2001
Message-Id: <28a8c2b2ba9e6af4da016beae1e6679015f6d591.1335361915.git.minovotn@redhat.com>
In-Reply-To: <ce1a7d1539a0b4b36555d1035257f57af7ae8478.1335361915.git.minovotn@redhat.com>
References: <ce1a7d1539a0b4b36555d1035257f57af7ae8478.1335361915.git.minovotn@redhat.com>
From: Paolo Bonzini <pbonzini@redhat.com>
Date: Tue, 24 Apr 2012 14:01:34 +0200
Subject: [PATCH 7/8] block: remove duplicate check in qmp_transaction

RH-Author: Paolo Bonzini <pbonzini@redhat.com>
Message-id: <1335276095-25813-8-git-send-email-pbonzini@redhat.com>
Patchwork-id: 39432
O-Subject: [RHEL 6.3 qemu-kvm PATCH 7/8] block: remove duplicate check in qmp_transaction
Bugzilla: 813862
RH-Acked-by: Marcelo Tosatti <mtosatti@redhat.com>
RH-Acked-by: Kevin Wolf <kwolf@redhat.com>
RH-Acked-by: Jeffrey Cody <jcody@redhat.com>

Bugzilla: 813862

RHEL-only, pointed out by Laszlo.
---
 blockdev.c |    5 -----
 1 file changed, 5 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 blockdev.c |    5 -----
 1 files changed, 0 insertions(+), 5 deletions(-)

diff --git a/blockdev.c b/blockdev.c
index 1634a2d..aea72e4 100644
--- a/blockdev.c
+++ b/blockdev.c
@@ -959,11 +959,6 @@ void qmp_transaction(BlockdevActionList *dev_list, Error **errp)
             goto delete_and_fail;
         }
 
-        if (!bdrv_is_inserted(states->old_bs)) {
-            error_set(errp, QERR_DEVICE_HAS_NO_MEDIUM, device);
-            goto delete_and_fail;
-        }
-
         if (full && mode != NEW_IMAGE_MODE_EXISTING) {
             assert(format && drv);
             bdrv_get_geometry(states->old_bs, &size);
-- 
1.7.7.6

